README.md

```diff
@@ -0,0 +1,28 @@
+# pieces-voice-chat
+
+This is a simple voice chat CLI application that allows users to communicate with [pieces](https://pieces.app) other using their voice.
+
+## Installation
+
+1. Clone the repository and switch to the project directory
+```bash
+git clone https://github.com/mason-at-pieces/pieces-voice-chat.git
+cd pieces-voice-chat
+```
+
+2. Install the dependencies
+```bash
+npm install
+```
+
+3. Run the application
+```bash
+node src/index.js
```

Comment:
Also let's make this `npm run start`.

This will also require updating the `package.json` script command from `run` to `start` 👍